tests/backup/backup_helper.go

```diff
@@ -3996,6 +3998,20 @@ func IsClusterPresent(clusterName string, ctx context.Context, orgID string) (bo
 	return false, nil
 }
 
+// GetConfigObj reads the configuration file and returns a Config object.
+func GetConfigObj() (*backup.Config, error) {
```

Comment:
Can we rename `Config` struct to `BackupCloudConfig`?
Also change the doc string for it

Reply:
done